packages/salesforcedx-vscode-core/package.json

```diff
@@ -32,6 +32,7 @@
     "@salesforce/salesforcedx-utils-vscode": "56.15.0",
```

Comment:
I think the package-lock.json in the salesforcedx-vscode-core folder above was added by mistake.